service/src/test/java/greencity/service/ubs/UBSManagementServiceImplTest.java

```diff
@@ -2171,4 +2172,14 @@ void updateOrderExportDetailsWhenDeliverToIsNull() {
         verify(orderRepository, times(1)).save(order);
         verify(eventService, times(1)).saveEvent(expectedHistoryEvent, employee.getEmail(), order);
     }
+
+    @Test
+    void getOrderCancellationReasonTest() {
+        OrderCancellationReasonDto cancellationReasonDto = ModelUtils.getCancellationDto();
+        Order order = ModelUtils.getOrderTest();
+        when(orderRepository.findById(anyLong())).thenReturn(Optional.ofNullable(order));
```

Comment:
please do not use `anyLong()`
It is better to use expected `orderId`
And please add `verify`